docs/designers-developers/developers/tutorials/block-tutorial/introducing-attributes-and-editable-fields.md

```diff
@@ -120,6 +120,8 @@ Earlier examples used the `createElement` function to create DOM nodes, but it's
 
 The `RichText` component can be considered as a super-powered `textarea` element, enabling rich content editing including bold, italics, hyperlinks, etc. It is not too much unlike the single editor region of the legacy post editor, and is in fact powered by the same TinyMCE library.
 
+To use the `RichText` component, add `wp-editor` to the array of registered script handles when calling `wp_register_script`.
```

Comment:
It would be awesome to get a code example here, if possible (presumably just taking the example from the first page in the tutorial and updating it)

Reply:
@greenhornet79 Could you put this code sample below the "To use the `RichText` component" sentence so it's a little clearer what the code sample is explaining?

I think also instead of the entire block registration function, it would be easier to just show the wp_register_script segment, calling out the change:

```php
wp_register_script(
	'gutenberg-boilerplate-es5-step03',
	plugins_url( 'step-03/block.js', __FILE__ ),
	array(
		'wp-blocks',
		'wp-element'
		'wp-editor', // Note the addition of wp-editor to the dependencies
	)
);
```